helm/prometheus-rules/templates/kaas/tenet/alerting-rules/apiserver.workload-cluster.rules.yml

```diff
@@ -35,7 +36,7 @@ spec:
       annotations:
         description: '{{`Kubernetes API Server {{ $labels.cluster_id }} having admission webhook errors (webhook: {{ $labels.name }}).`}}'
         opsrecipe: apiserver-admission-webhook-errors/
-      expr: rate(apiserver_admission_webhook_rejection_count{cluster_type="workload_cluster", error_type=~"calling_webhook_error|apiserver_internal_error"}[5m]) > 0
+      expr: rate(apiserver_admission_webhook_rejection_count{cluster_type="workload_cluster",error_type=~"calling_webhook_error|apiserver_internal_error",name!~"validate\\.kyverno\\.svc-fail|mutate\\.kyverno\\.svc-fail"}[5m]) > 0
```

Comment:
I would assume a single escape slash is needed here.

Reply:
I tested it on a CAPA and when I tried 

<img width="1237" alt="image" src="https://github.com/user-attachments/assets/3a35bf4e-4b55-4ef2-ae5e-2bf2493aeca8" />

it was not working but when I used double slash I got 

<img width="1233" alt="image" src="https://github.com/user-attachments/assets/e7c97ee9-19ad-47c3-b1eb-1b711c5c6614" />

Reply:
Ok, if that was tested fine with me then.